src/SavedSearch_Alert.php

```diff
@@ -65,7 +65,7 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
 
        // can exists for template
         if (
-            ($item->getType() == 'SavedSearch')
+            ($item instanceof self)
```

Comment:
`self` is `SavedSearch_Alert`
```suggestion
            ($item instanceof SavedSearch)
```